server/controllers/request.controller.js

```diff
@@ -0,0 +1,61 @@
+const RequestInfo = require('../models/request.model.js');
+
+exports.getRequests = async (req, res) => {
+    try {
+        const requestInfos = await RequestInfo.find();
+        res.status(200).json(requestInfos);
+    } catch (error) {
+        res.status(404).json( {message: error.message} );
```

Comment:
Assuming 404 not found for any errors thrown from the mongodb is not very helpful for debugging, it's best to just have 500 errors for this area.

Reply:
Mongoose won't throw an error is no requests are found